pages/faqs/faq-inspector.mdx

```diff
@@ -50,3 +50,15 @@ Reach out to support@avo.app or on Intercom, we would love to help!
 ### What kind of alerts can I set, if any?
 
 You can receive email alerts when new issues are detected. Head to the Issues dashboard on avo.app to configure email alerts.
+
+### When will the "bubble" from the In-App Inspector (aka Avo Debuggers) be visible in my app?
+
+The "bubble" of the [In-App Inspector (aka Avo Debuggers)](https://www.avo.app/docs/explore-tracking-plan/start-using-visual-debuggers) is visible when [the debugger is initialized](https://github.com/avohq/react-native-analytics-debugger) in a `dev` environment. Read more about the 
+
+### If I ship the Inspector SDK to prod, will the In-App Inspector bubble be visible in prod?
+
+Nope! 
+
+First of all, [setting up the Inspector SDK](https://www.avo.app/docs/implementation/setup-inspector-sdk) does not automatically activate the [In-App Inspector (aka Avo Debuggers)](https://www.avo.app/docs/explore-tracking-plan/start-using-visual-debuggers). Instead, the Inspector SDK activates the [Inspector Dashboard](https://www.avo.app/inspector) so you get ongoing data observability and can start figuring out what is wrong with tracking 😅
+
+Second of all: The "bubble" is only on in `dev`. (More in above answer).
```

Comment:
If you set the Inspector SDK to dev mode, the visual debugger will pop up, at least for the platforms that have them baked together.

I feel like we could simplify this answer down to the "second of all" part. We never pop up the bubble if you initialize the Inspector SDK with the env set to `prod`.

Reply:
Needs a little tweak, now Android and iOS Inspector SDKs activate the bubble in all environments except prod.